packages/config/config/devices/0x0039/templates/honeywell_template.json

```diff
@@ -0,0 +1,683 @@
+		]
+	},
+	"reversing_valve": {
+		"label": "Reversing Valve",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 1,
+		"defaultValue": 0,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "O/B on Cool",
+				"value": 0
+			},
+			{
+				"label": "O/B on Heat",
+				"value": 1
+			}
+		]
+	},
+	"cool_stages": {
+		"label": "Cool Stages",
+		"$import": "#base_hvac_stages_2"
+	},
+	"heat_stages": {
+		"label": "Heat Stages",
+		"description": "Heat stages or auxiliary/emergency stages",
+		"$import": "#base_hvac_stages_2"
+	},
+	"auxiliary_emergency_control": {
+		"label": "Auxiliary/Emergency Control",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 1,
+		"defaultValue": 0,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Both auxiliary and emergency",
+				"value": 0
+			},
+			{
+				"label": "Either auxiliary or emergency",
+				"value": 1
+			}
+		]
+	},
+	"auxiliary_heat_type": {
+		"label": "Auxiliary Heat Type",
+		"$import": "#base_aux_em_heat_type"
+	},
+	"emergency_heat_type": {
+		"label": "Emergency Heat Type",
+		"$import": "#base_aux_em_heat_type"
+	},
+	"fossil_kit_control": {
+		"label": "Fossil Kit Control",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 1,
+		"defaultValue": 0,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Thermostat",
+				"value": 0
+			},
+			{
+				"label": "External",
+				"value": 1
+			}
+		]
+	},
+	"auto_changeover": {
+		"label": "Auto Changeover",
+		"$import": "../../templates/master_template.json#base_enable_disable"
+	},
+	"auto_differential": {
+		"label": "Auto Differential",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 5,
+		"defaultValue": 0,
+		"unit": "°F"
+	},
+	"high_cool_stage_finish": {
+		"label": "High Cool Stage Finish",
+		"$import": "../../templates/master_template.json#base_enable_disable"
+	},
+	"high_heat_stage_finish": {
+		"label": "High Heat Stage Finish",
+		"$import": "../../templates/master_template.json#base_enable_disable"
+	},
+	"auxiliary_heat_droop": {
+		"label": "Auxiliary Heat Droop",
+		"description": "Allowable range: 2-15",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 15,
+		"defaultValue": 0,
+		"unit": "°F",
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Comfort",
+				"value": 0
+			},
+			{
+				"label": "2",
+				"value": 2
+			},
+			{
+				"label": "3",
+				"value": 3
+			},
+			{
+				"label": "4",
+				"value": 4
+			},
+			{
+				"label": "5",
+				"value": 5
+			},
+			{
+				"label": "6",
+				"value": 6
+			},
+			{
+				"label": "7",
+				"value": 7
+			},
+			{
+				"label": "8",
+				"value": 8
+			},
+			{
+				"label": "9",
+				"value": 9
+			},
+			{
+				"label": "10",
+				"value": 10
+			},
+			{
+				"label": "11",
+				"value": 11
+			},
+			{
+				"label": "12",
+				"value": 12
+			},
+			{
+				"label": "13",
+				"value": 13
+			},
+			{
+				"label": "14",
+				"value": 14
+			},
+			{
+				"label": "15",
+				"value": 15
+			}
+		]
```

Comment:
```suggestion
		"allowManualEntry": true,
		"options": [
			{
				"label": "Comfort",
				"value": 0
			}
		]
```

Reply:
The reason I used the options was is because the value "1" is not a valid option. Should I just remove the units property and add that to each of the options instead?

Reply:
Usually when something like this comes up (e.g. 0 and 5-600), we mention the allowable range in the description like you did. I don't feel very strongly here, since these aren't tons of options, so you can also just move the unit to the labels.